app/models/restroom.rb

```diff
@@ -3,27 +3,29 @@
 # (accessible) is coded by 1
 
 class Restroom < ApplicationRecord
-
   include PgSearch::Model
-  pg_search_scope :search, against: {
-    :name => 'A',
-    :street => 'B',
-    :city => 'C',
-    :state => 'D',
-    :comment => 'B',
-    :directions => 'B',
-    :country => 'D',
-  },
-  using: {tsearch: {dictionary: "english"}},
-  ignoring: :accents
+  pg_search_scope(
+    :search,
+    against: {
+      name: 'A',
+      street: 'B',
+      city: 'C',
+      state: 'D',
+      comment: 'B',
+      directions: 'B',
+      country: 'D'
+    },
+    using: { tsearch: { dictionary: "english" } },
+    ignoring: :accents
+  )
 
   validates :name, :street, :city, :state, presence: true
 
   geocoded_by :full_address
   after_validation :perform_geocoding
 
   reverse_geocoded_by :latitude, :longitude do |obj, results|
-    if geo = results.first
+    if geo == results.first
```

Comment:
I'm not 100% sure if this is the `==` was the desired effect for this.

Reply:
Hi,

I think this is meant to check if the assignment works (thereby checking that `results.first` exists and isn't undefined/null or what-have-you), and then use the gio object only if that's the case (only if it has become a copy of `results.first`).

I don't believe `geo` is defined anywhere else in the project, so the `==` likely makes `reverse_geocoded_by` fail to update the `obj` (object) passed to it. Sorry for the verbose explanation, but I'm explaining it to myself as I go as well. (Said so before, but I haven't edited the Ruby code much -- I have read much of it though several times, just not 100% understanding as I'm so-so with Ruby).

Reply:
Regardless of the intent or correctness, I presume this was copy-pasted and lightly modified from the `geocoder` gem's `README.md`:

https://github.com/alexreisner/geocoder/tree/v1.6.4#custom-result-handling